src/main/kotlin/gg/essential/elementa/markdown/MarkdownComponent.kt

```diff
@@ -45,6 +45,7 @@ class MarkdownComponent @JvmOverloads constructor(
     private var cursor: Cursor<*>? = null
     private var selection: Selection? = null
     private var canDrag = false
+    private var initialLayout = false
```

Comment:
I'd be inclined to call this something like `needsInitialLayout` and invert its usages. To me, it wasn't immediately obvious from the name whether this indicates that the component does or doesn't need to be laid out.